sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerProperties.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.aad.webapi;
+
+import com.azure.spring.aad.implementation.constants.AADTokenClaim;
+import com.azure.spring.aad.implementation.constants.AuthorityPrefix;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.StringUtils;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Configuration properties for Azure Active Directory B2C.
+ */
+@ConfigurationProperties("azure.activedirectory.resource-server")
```

Comment:
@yiliuTo , maybe should should use the new property prefix? Or support both new and old property prefix at the same time.